installer/util/BUILD.gn

```diff
@@ -10,3 +15,25 @@ source_set("util") {
     "//components/version_info",
   ]
 }
+
+if (is_win) {
+  source_set("vpn_utils") {
```

Comment:
I would put these in `installer/win/util`

Reply:
Fixed in https://github.com/brave/brave-core/pull/20754/commits/d79ae7e78fe451fb5570ede658025365d0cb5a10